data/transactions/transaction.go

```diff
@@ -726,9 +726,9 @@ func ProgramVersion(bytecode []byte) (version uint64, length int, err error) {
 // matching versions between approval and clearstate.
 const syncProgramsVersion = 6
 
-// CheckContractVersions ensures that for v6 and higher two programs are version
```

Comment:
@jannotti see if this wording works or feel free to adjust inline on the branch.